independent-projects/enforcer-rules/src/main/resources/enforcer-rules/quarkus-banned-dependencies.xml

```diff
@@ -51,8 +51,6 @@
                 <exclude>org.wildfly.client:wildfly-client-config</exclude>
                 <exclude>org.jboss.marshalling:jboss-marshalling-osgi</exclude>
                 <exclude>org.jboss.spec.javax.servlet:jboss-servlet-api_3.1_spec</exclude>
-                <!-- The API is packaged by the implementation-->
-                <exclude>jakarta.json:jakarta.json-api</exclude>
```

Comment:
This is a no no. It will have to wait for the Jakarta merge. Doing this, you are mixing Javax and Jakarta stuff.

That being said, let's get this properly reviewed and ready for when we do the merge.

Reply:
@gsmet I thought as artifact location is different and package is different the two versions can co-exist. But if you think it should wait for the migration I'm OK with it.
Let's review it and merge it later.
Do we have a label for such thing or is it the only PR that is blocked untill the migration ?